src/server/server.cc

```diff
@@ -1163,7 +1163,11 @@ void Server::GetInfo(const std::string &ns, const std::string &section, std::str
 
     if (section_cnt++) string_stream << "\r\n";
     string_stream << "# Keyspace\r\n";
-    string_stream << "# Last scan db time: " << std::put_time(&last_scan_tm, "%a %b %e %H:%M:%S %Y") << "\r\n";
+    if (last_scan_time == 0) {
+      string_stream << "# WARN: DB not scan, run `dbsize scan` command first.\r\n";
+    } else {
+      string_stream << "# Last scan DB time: " << std::put_time(&last_scan_tm, "%a %b %e %H:%M:%S %Y") << "\r\n";
```

Comment:
```suggestion
      string_stream << "# WARN: DBSIZE SCAN never performed yet\r\n";
    } else {
      string_stream << "# Last DBSIZE SCAN time: " << std::put_time(&last_scan_tm, "%a %b %e %H:%M:%S %Y") << "\r\n";
```

Reply:
Not all users will read the documentation, so I suggest directly writing `dbsize scan` command in the comments.

Reply:
it's written in the comment.

Reply:
Oh, my mistake. I misunderstood `DBSIZE SCAN` as a noun instead of a command.